src/test/java/org/opensearch/knn/bwc/KNNBackwardsCompatibilityIT.java

```diff
@@ -0,0 +1,193 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import org.apache.http.util.EntityUtils;
+import org.opensearch.knn.KNNResult;
+import org.opensearch.knn.index.KNNQueryBuilder;
+import org.opensearch.knn.KNNRestTestCase;
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.junit.Assert;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+
+public class KNNBackwardsCompatibilityIT extends KNNRestTestCase {
+    private static final ClusterType CLUSTER_TYPE = ClusterType.parse(System.getProperty("tests.rest.bwcsuite"));
```

Comment:
CLUSTER_TYPE throws exception, i don't think it is safe to declare as static. Either, create a method or don't throw exception

Reply:
Updated